app/styles/_base.scss

```diff
@@ -81,3 +81,14 @@ a {
     }
   }
 }
+
+.section-title {
+  color: $color-9;
+  display: block;
+  font-size: $font-size-xx-big;
+  font-weight: $font-weight-normal;
+  line-height: 1.05;
+  margin: 60px 0 18px;
+  max-width: 700px;
+  width: 100%;
+}
```

Comment:
This was almost duplicated from blog's c-list-posts.scss. It should probably be a base style, not at component level.